aptos-move/framework/aptos-framework/sources/aptos_account.spec.move

```diff
@@ -121,105 +131,133 @@ spec aptos_framework::aptos_account {
         ensures global<DirectTransferConfig>(addr).allow_arbitrary_coin_transfers == allow;
     }
 
-    spec batch_transfer(source: &signer, recipients: vector<address>, amounts: vector<u64>) {
+    spec batch_transfer(
+        source: &signer,
+        recipients: vector<address>,
+        amounts: vector<u64>
+    ) {
         //TODO: Can't verify the loop invariant in enumerate
         pragma verify = false;
         let account_addr_source = signer::address_of(source);
         let coin_store_source = global<coin::CoinStore<AptosCoin>>(account_addr_source);
         let balance_source = coin_store_source.coin.value;
 
-        requires forall i in 0..len(recipients):
-            recipients[i] != account_addr_source;
-        requires exists i in 0..len(recipients):
-            amounts[i] > 0;
+        requires forall i in 0..len(recipients): recipients[i] != account_addr_source;
+        requires exists i in 0..len(recipients): amounts[i] > 0;
 
         // create account properties
         aborts_if len(recipients) != len(amounts);
-        aborts_if exists i in 0..len(recipients):
-                !account::exists_at(recipients[i]) && length_judgment(recipients[i]);
-        aborts_if exists i in 0..len(recipients):
-                !account::exists_at(recipients[i]) && (recipients[i] == @vm_reserved || recipients[i] == @aptos_framework || recipients[i] == @aptos_token);
-        ensures forall i in 0..len(recipients):
-                (!account::exists_at(recipients[i]) ==> !length_judgment(recipients[i])) &&
-                    (!account::exists_at(recipients[i]) ==> (recipients[i] != @vm_reserved && recipients[i] != @aptos_framework && recipients[i] != @aptos_token));
+        aborts_if exists i in 0..len(recipients): !account::exists_at(recipients[i]) && length_judgment(
+            recipients[i]
+        );
+        aborts_if exists i in 0..len(recipients): !account::exists_at(recipients[i]) && (
+            recipients[i] == @vm_reserved || recipients[i] == @aptos_framework || recipients[
+                i
```

Comment:
This `i` seems to be in a separate line?

Reply:
fixed